internal/app/cf-terraforming/cmd/generate.go

```diff
@@ -43,36 +43,36 @@ func generateResources() func(cmd *cobra.Command, args []string) {
 		zoneID = viper.GetString("zone")
 		accountID = viper.GetString("account")
 
-		tmpDir, err := ioutil.TempDir("", "tfinstall")
-		if err != nil {
-			log.Fatal(err)
-		}
-		defer os.RemoveAll(tmpDir)
+		var execPath, workingDir string
+		workingDir = viper.GetString("terraform-install-path")
+		execPath = viper.GetString("terraform-path")
```

Comment:
this flag name is a little too vague as to what it actually does. how about `terraform-binary-path` instead?

```suggestion
		execPath = viper.GetString("terraform-binary-path")
```

Reply:
Yeah, that's much better!